tests/test_api/Tests/Verificators/BootSequenceVerificator.py

```diff
@@ -0,0 +1,35 @@
+from Tests.Verificators.OutputBaseVerificator import *
+from Tests.Utils.JsonUtils import *
+from testfixtures import *
+
+import hashlib
+
+def verifyContractHash(output, contractName, contractsManager):
+    path = contractsManager.getWasmFilePath(contractName)
+    file = open(path, 'rb')
+    hash = hashlib.sha256()
+    hash.update(file.read())
+    file.close()
+    verifyOutputContains(output, hash.hexdigest())
+
+def verifyContractAbi(output, contractName, contractsManager):
+    path = contractsManager.getAbiFilePath(contractName)
+    file = open(path, 'r')
+    abiFromFileString = file.read()
+    file.close()
+    abiFromNodeString = "".join(output)
+
+    abiFromNode = jsonToPython(abiFromNodeString)
+    removeAdditionalFields(abiFromNode)
```

Comment:
Why do you remove `scope_type` attribute? it is more correct to repeat the `nodeos` logic: add this attribute with an empty value if it is absent.
As well as `error_messages`.